components/transactions/TransactionItem.vue

```diff
@@ -261,6 +292,17 @@ export default {
       }
       return null
     },
+    classId() {
```

Comment:
`classId` seems too generic
```suggestion
    nftClassId() {
```